build_tools/benchmarks/definitions/iree.py

```diff
@@ -0,0 +1,80 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import List
+
+from definitions.common import DeviceArchitecture, DevicePlatform, DeviceSpec, Model, ModelInputData
```

Comment:
Couldn't this just be "from common"?

Reply:
Changed to relative import `from .common`. `from common` doesn't work if the module is imported from other directory.